src/index.js

```diff
@@ -20,8 +23,13 @@ const DEFAULT_REPLACEMENT = '--REDACTED--';
 
 module.exports = (
   { blacklist = [], whitelist = [] } = {},
-  { replacement = () => DEFAULT_REPLACEMENT, trim = false } = {}
+  { replacement = () => DEFAULT_REPLACEMENT, strategy = REDACT } = {}
 ) => {
+  if (![REDACT, TRIM, TRIM_AND_LIST].includes(strategy)) {
+    // eslint-disable-next-line
```

Comment:
May not be required with the below changes.

Reply:
Done